docs/apache-airflow/howto/define-extra-link.rst

```diff
@@ -99,7 +99,11 @@ tasks using :class:`~airflow.providers.amazon.aws.transfers.gcs_to_s3.GCSToS3Ope
       operators = [GCSToS3Operator]
 
       def get_link(self, operator: BaseOperator, *, ti_key: TaskInstanceKey):
-          return "https://s3.amazonaws.com/airflow-logs/{dag_id}/{task_id}/{run_id}".format(
+          # Invalid bucket name because upper case letters are used
+          # This will not be a valid bucket in any region
+          bucket_name = "AirflowBucket"
+          return "https://s3.amazonaws.com/airflow-logs/{bucket_name}/{dag_id}/{task_id}/{run_id}".format(
```

Comment:
I'm good with this, but I might prefer to have something obviously not a valid name 🤔

Reply:
Yeah, made it even more invalid now. WDYT?

Reply:
Looks great 🙌